static_inventories/forkhead_cluster.yml

```diff
@@ -28,29 +28,27 @@ all:
       hosts:
         forkhead:
           slurm_sockets: 1
-          slurm_cores_per_socket: 4    # 64 after being resized
+          slurm_cores_per_socket: 4
           slurm_real_memory: 15884
           slurm_local_disk: 0
           slurm_features: 'tmp12'
           slurm_ethernet_interfaces:
             - eth0
-            - eth1
     compute_vm:
       children:
         regular:  # Must be item from {{ slurm_partitions }} variable defined in group_vars/{{ stack_name }}/vars.yml
           hosts:
             forkhead:
           vars:
             slurm_sockets: 1
-            slurm_cores_per_socket: 4    # 64 after being resized
+            slurm_cores_per_socket: 4
             slurm_real_memory: 15884
-            slurm_max_cpus_per_node: "{{ slurm_sockets * slurm_cores_per_socket - 8 }}"
-            slurm_max_mem_per_node: "{{ slurm_real_memory - 8 * 2048 }}"
+            slurm_max_cpus_per_node: "{{ slurm_sockets * slurm_cores_per_socket - 2 }}"
+            slurm_max_mem_per_node: "{{ slurm_real_memory - 8 * 512 }}"
```

Comment:
When the amount of RAM we want to reserve for the OS and everything else that is not a Slurm job does not depend on the amount of cores, it does not make sense to multiply values here. Instead of 
`            slurm_max_mem_per_node: "{{ slurm_real_memory - 8 * 512 }}"`
you can just use:
`            slurm_max_mem_per_node: "{{ slurm_real_memory - 4096 }}"`
That will fail though on a small VM, which only has 4096 GB RAM in total. 
`slurm_max_mem_per_node: "{{ slurm_real_memory - slurm_sockets * slurm_cores_per_socket * 512 }}"`
is a more flexible setup.